src/V3Stats.cpp

```diff
@@ -74,12 +87,40 @@ class StatsVisitor final : public VNVisitor {
         }
     }
 
+    void storageStats(AstNodeModule* nodep) {
+        const int minRAMbits = 128;  // Report all but trivially small RAMs
```

Comment:
Suggest move to a new // CONSTANTS section at top of file.
as otherwise kind of gets lost here.